render/container.go

```diff
@@ -0,0 +1,36 @@
+package render
```

Comment:
There is about 5 lines of 'real' code in this file - I'd drop the comment and stick this function in mapping.go